### ISSUE TITLE
Performance cluster control script no longer working with Uno

### ISSUE THREAD

**Issue by keith-turner** [apache/accumulo-testing#127]
Something has changed where the [cluster control script](https://github.com/apache/accumulo-testing/blob/main/conf/cluster-control.sh.uno) for uno no longer seems to work.  This script can be used by performance test.

The following commands are being run by the scripts.  These used to work and no longer work with the latest Uno.  Not sure what changed.  The goal of all of these commands is to avoid setting up hadoop and zookeeper from scratch between performance test.

```bash
  uno install accumulo
  uno run zookeeper
  uno run hadoop
  uno setup accumulo --no-deps
  uno stop accumulo --no-deps 
  uno start accumulo --no-deps 
```

**Comment by Manno15** [comment]
I did run into issues aswell. What is the specific issue you ran into? Mine was zookeeper version. I couldn't connect to zookeeper 3.6.1 but it worked when I reverted back to 3.4.14. Everything else seemed to work for me though.

**Comment by ctubbsii** [comment]
@keith-turner What specifically isn't working with the script?

Looking at the script, one immediate potential I can see is that the `put_server_code` function tries to copy its file argument into the `$ACCUMULO_HOME/lib/ext` directory. However, there is no reason for the script to assume that directory exists, or that it is configured in any way in Accumulo's class path. By default in 2.x, that directory isn't configured for anything. And, as far as I can tell, Uno isn't configured to add to its class path either.

Could this be the cause of the problem you are seeing?

**Comment by keith-turner** [comment]
> I did run into issues aswell. What is the specific issue you ran into? Mine was zookeeper version.

@Manno15 I did not  look into it earlier, because I was focused on getting the performance test working.  I just reverted the hack I made to the cluster control script to call `accumulo setup` and reran.  Looking into it seems like accumulo init is failing because it can not contact NN.

> Looking at the script, one immediate potential I can see is that the put_server_code function tries to copy its file argument into the $ACCUMULO_HOME/lib/ext directory. 

If that is not working it would not impact the PT I was working on in #125 because it has no server side code.  That function is called to make a jar available server side on the accumulo cluster.  If the Uno impl of the function is not working it could impact other perf test and may need to be improved.  It seems like it may have worked in the past for 2.0, but not sure if that we before or after 2.0 release.
